modules/directives/src/main/scala/scala/build/preprocessing/directives/Toolkit.scala

```diff
@@ -39,9 +39,17 @@ final case class Toolkit(
 }
 
 object Toolkit {
-  def resolveDependency(toolkitVersion: Positioned[String]) = toolkitVersion.map(version =>
-    val v = if version == "latest" then "latest.release" else version
-    dep"${Constants.toolkitOrganization}::${Constants.toolkitName}::$v,toolkit"
+  def resolveDependency(toolkitCoord: Positioned[String]) = toolkitCoord.map(coord =>
+    val tokens  = coord.split(':')
+    val version = tokens.last
+    val v       = if version == "latest" then "latest.release" else version
+    val flavor  = tokens.dropRight(1).headOption
+    val org = flavor match {
+      case Some("typelevel") => Constants.typelevelOrganization
+      case Some(org)         => org
+      case None              => Constants.toolkitOrganization
+    }
```

Comment:
Supports
```
//> using toolkit "typelevel:latest"
```

But there's a question what to do about:
```
//> using toolkit "foo:latest"
```
We could reject `foo` as an unknown toolkit. But instead what I did here was treat it as the group id. So then anyone can publish a toolkit artifact under their group id and it would Just Work.:tm: Thoughts?

Reply:
An alternative can be having a system similar to how sbt treats the testing frameworks: the most notorious are manually added to sbt while there's the support to use them, adding a minimal configuration (while waiting for sbt to upstream the particular testing framework)

Reply:
I think that should fine and allow us to not modify ScalaCLI every time someone wants to create or test out a new toolkit.

Reply:
Another interesting alternative is to somehow use GH org (e.g. `typelevel`, `scala`) instead of group id (`org.typelevel`, `org.scala-lang`). But this would be harder to implement :)

Reply:
Also, not everybody might use GH to distribute their toolkit, right?

Reply:
I was thinking the toolkit itself would still be on Maven, but somehow we use the GH orgs as the namespace to map to Maven coordinates.

e.g. `org.scala-lang` can just be `scala`, `org.typelevel` can just be `typelevel` (and we won't even need to hardcode it), etc.

Compare:

```scala
//> using toolkit "typelevel:latest"
```

With:

```scala
//> using toolkit "org.typelevel:latest"
```

And finally:
```scala
//> using dep "org.typelevel::toolkit::latest.release"
```